### PR TITLE
Adjust the backup naming convention and the health check to force PRO…

### PR DESCRIPTION
**Pull request by GeoffThorpeFT**
…D-DB into the naming - remove heroku names

### CODE REVIEW

File: cmdbv2api/settings.py [hunk 1]
```diff
@@ -39,10 +39,11 @@
 )
 
 DBBACKUP_STORAGE = 'dbbackup.storage.s3_storage'
+DBBACKUP_FILENAME_TEMPLATE = 'PROD-DB-{datetime}.{extension}'
```
Comment (lucas42) [inline cmdbv2api/settings.py]: I don't think we should hardcode the environment here.  Doesn't that defeat the purpose of having it?

Comment (captain-sysadmin) [inline cmdbv2api/settings.py]: it does make testing harder.

Comment (GeoffThorpeFT) [inline cmdbv2api/settings.py]: I dont think we have any variable at the moment that defines the name of the environment.  If we had I could included it.

Comment (captain-sysadmin) [inline cmdbv2api/settings.py]: if we have a prod flag it'd make testing for prod environment much easier....

**Comment by lucas42** [comment]
Looks fine.  Worth a try. 😃

**Pull request merged**